js/data/languages.js

```diff
@@ -1029,7 +1029,26 @@ function getTranslatedLangs(lang = app && app.localSettings &&
   });
 
   if (sort) {
-    translated = translated.sort((a, b) => a.name.localeCompare(b.name, lang));
+    translated = translated.sort((a, b) => {
```

Comment:
This was missed in a previous PR. Without this one of the settings tabs was bombing since JS does not recognize the transifex lang code - it needs to be standardized.